package.json

```diff
@@ -12,49 +12,48 @@
 	"author": "Microsoft and contributors",
 	"scripts": {
 		"build": "npm run policy-check && npm run layer-check && npm run build:genver && npm run build:compile && npm run lint && npm run build:docs",
-		"build:ci": "npm run build:genver && lerna run build:compile --stream",
-		"build:compile": "lerna run build:compile --stream",
-		"build:docs": "lerna run build:docs --stream --parallel",
+		"build:ci": "npm run build:genver && pnpm run -r --stream build:compile",
```

Comment:
I'd have to see exactly what it does but have we tried `--aggregate-output` instead of `--stream`? Based on the docs I'm not sure if it waits for _all_ parallel child processes to be done before printing their output (ugh) or if it does it as each of them finishes (yay). Maybe the latter is achieved by not providing either flag. If we can get output like that, I think it'd be very convenient for CI.

Reply:
Stream is good for CI since it ensures all information is visible (nothing is omitted or delayed). If you have one child process hung for example, stream lets you know the last thing the hung process did while --aggregate-output would never log anything for that process. There are benefits to both: I think which is better is pretty subjective.

This PR is about changing the task runner not adjusting the settings. It makes changes where needed (the CLI for pnpm works a bit differently than for lerna) to keep everything working, but when practical I'd like to keep the subjective decisions (ex: output format) the same as before.